.github/workflows/publish-canvas.yml

```diff
@@ -16,35 +16,29 @@ jobs:
         uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
 
       - name: Install Python 3.9
-        # very carefully use the tag when testing with act-cli on localhost, commit the hash version
-        # uses: actions/setup-python@v2
         uses: actions/setup-python@3105fb18c05ddd93efea5f9e0bef7a03a6e9e7df
         with:
           python-version: '3.9.x'
 
       - name: Install Poetry
-        # very carefully use the tag when testing with act-cli on localhost, commit the hash version
-        # uses: Gr1N/setup-poetry@v4
         uses: Gr1N/setup-poetry@462ac83c852d49e282a1233c4c24c5411696e7c7
 
-      - name: Install Python Dependencies
-        run: python ./eng/build.py install canvas-extractor
-
-      - name: Cache Dependencies
-        # very carefully use the tag when testing with act-cli on localhost, commit the hash version
-        # uses: actions/cache@v2
-        uses: actions/cache@0781355a23dac32fd3bac414512f4b903437991a
+      - name: Setup Dependency Caching
+        uses: actions/cache@c64c572235d810460d0d6876e9c705ad5002b353
```

Comment:
I didn't copy my dependency caching improvements to the publish files. Need to do that before pulling out of draft.

Reply:
done